packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -90,6 +90,16 @@ abstract class SelectionHandler implements ValueListenable<SelectionGeometry> {
   /// Return `null` if nothing is selected.
   SelectedContent? getSelectedContent();
 
+  /// Gets the list of selections in this object.
+  ///
+  /// The order of this list follows the same
+  /// order as the [Selectable]s contained under
+  /// this [SelectionHandler].
+  ///
+  /// Return a list of [SelectedContentRange.empty]
```

Comment:
nit: Returns

Reply:
Also: why doesn't it return an empty list?

Reply:
We need to be able to add up the `contentLength`s of `Selectable`s even if they are not currently selected. For example if we have `[A,B,C,D]`, and only `[B,C,D]` are selected, I still need the length of `A` to calculate the offsets.